skiko/build.gradle.kts

```diff
@@ -354,6 +368,75 @@ project.tasks.register<Exec>("wasmCompile") {
     outputs.files(outJs, outWasm)
 }
 
+fun List<String>.findAllFiles(suffix: String): List<String> = this
+    .map { File(it) }
+    .flatMap { it.walk().toList() }
+    .map { it.absolutePath }
+    .filter { it.endsWith(suffix) }
+
+
+// Very hacky way to compile native bridges and add the
+// resulting object files into the final native klib.
+project.tasks.register<Exec>("nativeBridgesCompile") {
+    dependsOn(skiaDir)
+    val inputDirs = listOf(
+        "$projectDir/src/macosX64Main/cpp/generated",
+        "$projectDir/src/macosX64Main/cpp/common",
+        "$projectDir/src/commonMain/cpp/common"
+    )
+    val outDir = "$buildDir/nativeBridges/obj/$target"
+    val srcs = inputDirs
+        .findAllFiles(".cc")
+        .toTypedArray()
+    val outs = srcs
+        .map { it.substringAfterLast("/") }
+        .map { File(it).nameWithoutExtension }
+        .map { "$outDir/$it.o" }
+        .toTypedArray()
+
+    workingDir = File(outDir)
+    commandLine = listOf(
+        "clang++",
+        *targetArch.clangFlags,
+        "-mmacosx-version-min=10.13",
```

Comment:
Better move that to argetArch.clangFlags

Reply:
moved to targetOs.clangFlags